falcon/request_helpers.py

```diff
@@ -15,9 +15,11 @@
 """Utilities for the Request class."""
 
 from http import cookies as http_cookies
-import io
 import re
 
+# TODO: BoundedStream  import here is for backwards-compatibility
+# and it should be removed in Falcon 4.0
+from falcon.stream import BoundedStream, Body  # NOQA
```

Comment:
I guess we could use module lever `__getattr__` here to raise a deprecation warning.
but maybe a follow-on change in 3.1 since I think there may be other places that could benefit from it

Reply:
I don't think this is a big deal since there are probably few, if any, apps that actually instantiate BoundedStream directly. But we can consider a 3.1 issue if you want to create one and assign it to that milestone. I just want to be sure that the `__getattr__` doesn't have any unexpected performance implications for Cython or PyPy (but I can't imagine why it would).